src/app/containers/StoryPromo/MediaIndicator.test.jsx

```diff
@@ -50,8 +77,18 @@ describe('Story Promo Media Indicator', () => {
     <MediaIndicator item={noDurationItem} />,
   );
 
+  shouldShallowMatchSnapshot(
```

Comment:
Should this be a `shallowMatch` or a null check?

Reply:
shallowMatch, as we do render a media indicator if the format is missing, just with no offscreen text :)